src/ui/server/graphql/queries/applications.ts

```diff
@@ -2,7 +2,12 @@ import gql from 'graphql-tag';
 
 const applicationsQuery = gql`
   query applications($accountId: ID!, $take: Int!, $skip: Int!) {
-    applications(where: { owner: { id: { equals: $accountId } } }, orderBy: { updatedAt: desc }, take: $take, skip: $skip) {
+    applications(
+      where: { AND: [{ owner: { id: { equals: $accountId } } }, { status: { not: { equals: "Abandoned" } } }] }
+      orderBy: { updatedAt: desc }
+      take: $take
+      skip: $skip
+    ) {
```

Comment:
@abhi-markan FYI these is the GQL syntax/variables RE discussions the other day 😎